templates/vue/src/components/modals/EmbedModal.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <b-modal
+    id="embed-modal"
+    data-qa="embed-modal"
+    :visible="show"
+    size="lg"
+    title="Create Embed"
+    body-class="p-0"
+    @hidden="$emit('close')"
+  >
+    <b-container fluid class="pt-3">
+      <b-form-row>
+        <b-col>
+          <b-form-group label="Width">
+            <b-form-input v-model="width" type="number"></b-form-input>
+          </b-form-group>
+        </b-col>
+        <b-col>
+          <b-form-group label="Height">
+            <b-form-input v-model="height" type="number"></b-form-input>
+          </b-form-group>
+        </b-col>
+      </b-form-row>
+      <b-form-group>
+        <b-form-checkbox v-model="showInfo">
+          Show information below the iFrame
+        </b-form-checkbox>
+      </b-form-group>
+      <b-form-group
```

Comment:
I think there might be a way to also copy the code automatically to their clipboard. We should give them the option to click a button to copy to clipboard.